port/win/env_win.cc

```diff
@@ -2071,8 +2077,13 @@ class WinEnv : public Env {
   std::vector<ThreadPool> thread_pools_;
   mutable std::mutex mu_;
   std::vector<std::thread> threads_to_join_;
+  static FnGetSystemTimePreciseAsFileTime GetSystemTimePreciseAsFileTime_;
+  static bool GetSystemTimePreciseAsFileTimeInitialized_;
 };
 
+FnGetSystemTimePreciseAsFileTime WinEnv::GetSystemTimePreciseAsFileTime_ = NULL;
```

Comment:
std::once() is already used when the env instantiated so you do not need to use it. This is usually redundant in C++11 because static initialization is now thread safe. It is just required for MSVC 13 still.